src/plugin/Component.vue

```diff
@@ -5,8 +5,9 @@
       :key="letter.key"
       :ref="`${baseRefName}${index}`"
       v-model.trim="letter.value"
+      :type="isPassword ? 'password' : 'tel'"
```

Comment:
It is a bad idea to pass computations into template. Best way is computed props.